launchdarkly/helper_test.go

```diff
@@ -0,0 +1,136 @@
+package launchdarkly
+
+import (
+	"errors"
+	"github.com/hashicorp/terraform/helper/schema"
+	"reflect"
+	"testing"
+)
+
+func TestParseCompositeID(t *testing.T) {
+	expectedErr := errors.New("error: Import composite ID requires two parts separated by colon, eg x:y")
+
+	testCases := []struct {
+		name      string
+		id        string
+		wantedP1  string
+		wantedP2  string
+		wantedErr error
+	}{
+		{
+			name:      "expected",
+			id:        "id:test",
+			wantedP1:  "id",
+			wantedP2:  "test",
+			wantedErr: nil,
+		},
+		{
+			name:      "with more than one separator",
+			id:        "id:test:id",
+			wantedP1:  "id",
+			wantedP2:  "test:id",
+			wantedErr: nil,
+		},
+		{
+			name:      "without separator",
+			id:        "test",
+			wantedP1:  "",
+			wantedP2:  "",
+			wantedErr: expectedErr,
```

Comment:
Since expectedErr is only used here you could put the definition right in this test case

Reply:
Or maybe we could defined it as a const since we have the same exact error below ?

Reply:
I prefer keeping it modular like that so we can easily add a test in the future if necessary what do you think